app/assets/javascripts/admin/addon/components/simple-list.js

```diff
@@ -47,10 +47,32 @@ export default Component.extend({
     this._onChange();
   },
 
+  @action
+  shift(operation, index) {
+    let futureIndex = index + operation;
+
+    if (futureIndex > this.collection.length - 1) {
+      futureIndex = 0;
+    } else if (futureIndex < 0) {
+      futureIndex = this.collection.length - 1;
+    }
+
+    const shiftedValue = this.collection[index];
+    this.collection.removeAt(index);
+    this.collection.insertAt(futureIndex, shiftedValue);
+
+    this._saveValues();
```

Comment:
```suggestion
    this._onChange();
```

In simple lists there is no `_saveValues`, the equivalent is `_onChange`.

Reply:
Done ✅  😄